signature/cose/cose.go

```diff
@@ -15,34 +26,524 @@ func init() {
 	}
 }
 
+// Protected Headers
+// https://github.com/notaryproject/notaryproject/blob/cose-envelope/signature-envelope-cose.md
+const (
+	headerKeyExpiry                       = "io.cncf.notary.expiry"
+	headerKeySigningTime                  = "io.cncf.notary.signingTime"
+	headerKeyAuthenticSigningTime         = "io.cncf.notary.authenticSigningTime"
+	headerKeySigningScheme                = "io.cncf.notary.signingScheme"
+	headerKeyVerificationPlugin           = "io.cncf.notary.verificationPlugin"
+	headerKeyVerificationPluginMinVersion = "io.cncf.notary.verificationPluginMinVersion"
+	headerKeyCrit                         = "crit"
+	headerKeyAlg                          = "alg"
+	headerKeyCty                          = "cty"
```

Comment:
Those labels are integers instead of strings.

Reply:
These are actually not used, will remove them. Cose is using go-cose's Headers instead.